.github/workflows/build.yml

```diff
@@ -105,6 +106,11 @@ jobs:
 
           mkdir build
 
+          # TODO signing prototype -- starting very small
+          shouldSign="$(jq <<<"$json" '.build.arch == "amd64" and any(.source.allTags[]; startswith("notary:"))')"
```

Comment:
To unpack this more explicitly: this will _only_ sign `amd64` builds of the `notary` image as-is.